### PR TITLE
Setup Files 

### PR DESCRIPTION
**Pull request by ajawashington**
## Description

Setup files for blah blah project with the PULL_REQUEST_TEMPLATE.md!

## Related Issue

#1 

## Motivation and Context

None.

## How Can This Be Tested?

Make sure the files exist, all should be empty but index.html.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by molliegoforth818** [comment]
looks like there is a pull request template.md file can you include that??

**Review by molliegoforth818 — APPROVED** [review]
Thanks for adding that! Everything else looks good!

File: index.html
Comment (robertncole1) [inline index.html:1]: PR states this file should not be empty

Comment (ajawashington) [reply index.html:1]: You are so correct, let me fix that!

**Review by TrinityChristiana — APPROVED** [review]
LGTM!

**Review by robertncole1 — APPROVED** [review]

**Pull request merged**